stablehlo/dialect/Base.td

```diff
@@ -174,13 +218,19 @@ def HLO_StaticDimensionTensor : RankedTensorOf<[HLO_DimensionValue], [HasStaticS
 def HLO_StaticShapeTensor : StaticShapeTensorOf<[
```

Comment:
remove this as it will not be used anywhere?

Reply:
it is being used for def HLO_StaticShapeTensorOrPerAxisQuantizedTensorOrToken